tests/e2e/v2store_deprecation_test.go

```diff
@@ -97,3 +97,22 @@ func TestV2DeprecationWriteOnlyNoV2Api(t *testing.T) {
 	_, err = proc.Expect("--enable-v2 and --v2-deprecation=write-only are mutually exclusive")
 	assert.NoError(t, err)
 }
+
+func TestV2DeprecationCheckCustomContentOffline(t *testing.T) {
+	e2e.BeforeTest(t)
+	dataDirPath := t.TempDir()
+
+	createV2store(t, dataDirPath)
+
+	assertVerifyCheckCustomContentOffline(t, dataDirPath)
+}
+
+func assertVerifyCheckCustomContentOffline(t *testing.T, dataDirPath string) {
+	t.Logf("Checking custom content in v2store - %s", dataDirPath)
+
+	proc, err := e2e.SpawnCmd([]string{e2e.BinDir + "/etcdutl", "check", "v2store", "--data-dir=" + dataDirPath}, nil)
+	assert.NoError(t, err)
+
+	_, err = proc.Expect("detected custom content in v2store")
+	assert.NoError(t, err)
+}
```

Comment:
Can we add one more case which doesn't have custom content in v2store?

Reply:
Added. PTAL